### PR TITLE
Add sequence contents graph

### PR DESCRIPTION
**Pull request by WojDob**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,13 +1,58 @@
 # Run this app with `python app.py` and
 # visit http://127.0.0.1:8050/ in your web browser.
-
 from io import StringIO
 from dash import Dash, html, dcc
 from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
```
Comment (taboniek) [inline app.py:6]: How is your score in pylint?

Comment (WojDob) [reply app.py:6]: good

Comment (WojDob) [reply app.py:6]: good enough*

**Review by taboniek — APPROVED** [review]
good enough

**Pull request merged**